src/Atc.Azure.Options/Extensions/ConfigurationExtensions.cs

```diff
@@ -10,6 +10,9 @@ namespace Atc.Azure.Options.Extensions
 {
     public static class ConfigurationExtensions
     {
+        public static void ConfigureKeyVaultForApi(this IConfigurationBuilder config)
+            => ConfigureKeyVaultForApi(config, new NamingProvider());
```

Comment:
Directly new'ing this up here might make it harder to use a custom NamingProvider.

Reply:
There is an overload where you can specify a different NamingProvider. Is that not sufficient?

Reply:
This is just an overload. 
There is still the:
```
ConfigureKeyVaultForApi(
            this IConfigurationBuilder config,
            INamingProvider namingProvider,
            string environmentOptionsSectionName = nameof(EnvironmentOptions),
            string namingOptionsSectionName = nameof(NamingOptions))
```
Or did i misunderstand something ?

Reply:
Yes, missed that